rq/job.py

```diff
@@ -1386,6 +1428,16 @@ def execute_failure_callback(self, death_penalty_class: Type[BaseDeathPenalty],
             logger.exception(f'Job {self.id}: error while executing failure callback')
             raise
 
+    def execute_stopped_callback(self, death_penalty_class: Type[BaseDeathPenalty]):
+        """Executes stopped_callback with possible timeout"""
+        logger.debug('Running stopped callbacks for %s', self.id)
+        try:
+            with death_penalty_class(self.stopped_callback_timeout, JobTimeoutException, job_id=self.id):
+                self.stopped_callback(self, self.connection)
```

Comment:
Do you feel like getting 100% coverage? If yes, this can be covered by mocking stopped_callback to raise an exception.

Although I’m already quite happy with this patch as it is (this is optional 😀)